src/components/carousel/Carousel.jsx

```diff
@@ -0,0 +1,76 @@
+import carousel from "./carousel.module.css";
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+
+function SliderLabels({ activeSlide, setActiveSlide, inputNum }) {
+  return (
+    <>
+      {inputNum.map((item) => {
+        return (
+          <label
+            key={uuid()}
+            htmlFor={item}
+            onClick={() => {
+              setActiveSlide(item);
+            }}
+            className={`${carousel["manual-btn"]}  
+              ${item === activeSlide ? carousel["active"] : "abcd"}`}
```

Comment:
Question - what is the significance of "abcd" in the else case?

Reply:
left it by mistake